jsk_robot_common/jsk_robot_startup/scripts/shutdown.py

```diff
@@ -24,22 +26,38 @@ class Shutdown(object):
 
     def __init__(self):
         rospy.loginfo('Start shutdown node.')
+        self.client_jp = actionlib.SimpleActionClient('/robotsound_jp', SoundRequestAction)
         rospy.Subscriber('shutdown', Empty, self.shutdown)
         rospy.Subscriber('reboot', Empty, self.reboot)
         self.shutdown_command = rospy.get_param(
             '~shutdown_command', '/sbin/shutdown -h now')
         self.reboot_command = rospy.get_param(
             '~reboot_command', '/sbin/shutdown -r now')
 
+    def speak(self, client, speech_text, lang=None):
```

Comment:
You can use `say` function.
https://github.com/jsk-ros-pkg/jsk_3rdparty/tree/bfc50811da593ffe906f678eb82f5a7f847a1c14/3rdparty/google_cloud_texttospeech#for-python-users

Reply:
You use `blocking=True`, so you don't have to write `client.actionclient.wait_for_server` and ` client.actionclient.wait_for_result()`
https://github.com/ros-drivers/audio_common/blob/f9ff6a29f790c115197c961a59ed3a6eaf157a60/sound_play/src/sound_play/libsoundplay.py#L338-L354

Reply:
Thank you. I updated it.